libs/designsystem/src/lib/components/chart/chart-helper.ts

```diff
@@ -1,4 +1,4 @@
-import { Options } from 'highcharts';
+import { Options, chart } from 'highcharts';
 import * as Highcharts from 'highcharts';
```

Comment:
Delete?
```suggestion
```

Reply:
No, needed for importing highchart modules below